griptape/drivers/prompt/openai_chat_prompt_driver.py

```diff
@@ -123,7 +123,10 @@ def _base_params(self, prompt_stack: PromptStack) -> dict:
 
         messages = self._prompt_stack_to_messages(prompt_stack)
 
-        params["max_tokens"] = self.max_output_tokens(self.prompt_stack_to_string(prompt_stack))
+        if isinstance(self.tokenizer, OpenAiTokenizer):
```

Comment:
Should we change the tokenizer type to `OpenAiTokenizer` instead?

Reply:
This came from https://github.com/griptape-ai/griptape/pull/537